MatrixSDKTests/MXSpaceServiceTest.swift

```diff
@@ -48,6 +48,39 @@ class MXSpaceServiceTest: XCTestCase {
         }
     }
     
+    private func createSpaces(with spaceService: MXSpaceService, spaceNames: [String], completion: @escaping (_ spaces: MXResponse<[MXSpace]>) -> Void) {
+        
+        let dispatchGroup = DispatchGroup()
+                                                            
+        var spaces: [MXSpace] = []
+                        
+        for spaceName in spaceNames {
+            
+            dispatchGroup.enter()
+            
+            spaceService.createSpace(withName: spaceName, topic: nil, isPublic: true) { (response) in
+                                
+                switch response {
+                case .success(let space):
+                    spaces.append(space)
+                case .failure(let error):
+                    XCTFail("Fail to create space named: \(spaceName) with  error: \(error)")
+                }
+
+                dispatchGroup.leave()
+            }
+        }
+        
+        dispatchGroup.notify(queue: .main) {
+            
+            if spaces.count == spaceNames.count {
+                completion(.success(spaces))
+            } else {
+                XCTFail("Fail to create spaces")
```

Comment:
We should return an error to avoid blocking the test suite. See other comments in this file.